LoopStudio/LoopTracker
# Feature/48 es6 setup

## Review Process

#### What does this PR do?
Adds support for ES6 and JS linting to the project
#### Where should the reviewer start?
`app/javascript/packs/`
#### How should this be manually tested?
Check for the previously implemented features to not have been broken by this update
#### Any background context you want to provide?

...

#### What are the relevant issues?
Closes #48 
#### Screenshots (if appropriate)

...

#### Checklist:


---

## DevOps

#### Database changes

...

#### Deployment instructions (Specific order, ENV variables, etc...)
Now the project must be started by running `foreman start`
